From: David Rosca <nowrep@gmail.com>
Date: Fri, 22 Aug 2014 23:43:35 +0200
Subject: Fix compatibility with Linux 3.17

---
 acpi_call.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/acpi_call.c b/acpi_call.c
index 3025d97..2d9fb29 100644
--- a/acpi_call.c
+++ b/acpi_call.c
@@ -5,8 +5,8 @@
 #include <linux/version.h>
 #include <linux/proc_fs.h>
 #include <linux/slab.h>
+#include <linux/acpi.h>
 #include <asm/uaccess.h>
-#include <acpi/acpi.h>
 
 MODULE_LICENSE("GPL");
 
